.github/workflows/build.yml

```diff
@@ -60,6 +63,28 @@ jobs:
       if: runner.os == 'macOS'
       run: |
         brew install libuv swig pkgconfig jsoncpp
+    - uses: actions/checkout@v2
+    - name: Create directory for opentelemetry-cpp
+      run: mkdir -p "${OTelDir}"
+      shell: bash
+    - name: Checkout opentelemetry-cpp repo
+      uses: actions/checkout@v2
+      with:
+        repository: open-telemetry/opentelemetry-cpp
+        path: ${{env.OTelDir}}
+    - uses: actions/checkout@v2
```

Comment:
This looks redundant, duplicating the original proton checkout, with cleanup side effects.

Reply:
To be clearer, not sure why its highlighting all 4 lines...I just mean the last basic "- uses: actions/checkout@v2"

Reply:
Thanks, Done! Now it seems to be working. :)
 
And I just found out, that if we provide a directory beforehand then git clone will just dump data inside of it rather than creating a new directory. 
I manually created a directory for opentelemetry and thought git clone will also create a new subdirectory inside the one that I've created ( i.e. opentelemetry/opentelemetry-cpp). That's why it wasn't able to find CMakeLists file. :(